www/src/pages/docs/cli/integrations/css.md

```diff
@@ -309,20 +310,23 @@ export default defineConfig({
     css({
       filename: "tokens.css",
       exclude: [], // ex: ["beta.*"] will exclude all tokens in the "beta" top-level group
+      baseScheme: "light dark", // Optional: set base color-scheme
       modeSelectors: [
         {
           mode: "light",
           selectors: [
             "@media (prefers-color-scheme: light)",
             '[data-mode="light"]',
           ],
+          scheme: "light", // Optional: set color-scheme for this mode
         },
         {
           mode: "dark",
           selectors: [
             "@media (prefers-color-scheme: dark)",
             '[data-mode="dark"]',
           ],
+          scheme: "dark", // Optional: set color-scheme for this mode
```

Comment:
Question: what happens if the baseScheme is set to e.g. `only light` and scheme `dark` is passed? Or in other words, if an invalid scheme is passed?

Would it be useful to write some config validation?

Reply:
I don't have an opinion myself, and just figured I'd raise the question to see what you thought.

You have a solid reasoning informing your choice, I agree with you wrt the maintenance burden. I'm just not sure how hard it would be for users to debug a human error in setting the config without validation Vs having immediate error feedback.